src/redux/actions/experiments/createExperiment.js

```diff
@@ -28,23 +24,14 @@ const createExperiment = (
     ...experimentTemplate,
     id: experimentId,
     name: newExperimentName,
-    projectUuid,
+    projectUuid: experimentId,
     createdDate,
   };
 
-  let url;
-  let experimentToSend;
-
-  if (config.currentApiVersion === api.V1) {
-    experimentToSend = convertExperimentToApiV1Model(newExperiment);
+  const { id, name, description } = newExperiment;
+  const experimentToSend = { id, name, description };
```

Comment:
Why do you have this variable which makes id name and description into an object again? Can you not use `newExperiment` without creating `experimentToSend`?